Tasks/ManualIntervention/task.json

```diff
@@ -30,6 +30,14 @@
                 "rows": "10",
                 "maxLength": "500"
             }
+        },
+        {
+            "name": "emailRecipients",
+            "type": "identities",
```

Comment:
ensure this type is checked into mseng and deployed for both tfs and RM before releasing this task